source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+Parkinson’s Mouse Tutorial
+------------------------------------
+
+As an example of how to process data through QIIME 2, we will use a set of humanized murine fecal samples from `Sampson et al, 2016`_.
```

Comment:
How about:

```suggestion
This tutorial will demonstrate a "typical" QIIME 2 analysis, using samples of humanized murine fecal samples from `Sampson et al, 2016`_.
```

Reply:
Also, perhaps this sentence can just get merged with the next paragraph down (line 6)?